### DISCUSSION TITLE
replacement definition not being recognized

### DISCUSSION THREAD

**Discussion by travi** [renovatebot/renovate#14294] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

I've defined other replacements in [a separate config](https://github.com/form8ion/renovate-config/blob/master/js-replacements.json) that I extend in [my default config](https://github.com/form8ion/renovate-config/blob/8ca0cfcbbb383bfc51cb4c9f48026db25e05f2c0/default.json#L3). I've had success with this approach for [replacements that I've defined previously](https://github.com/form8ion/renovate-config/blob/8ca0cfcbbb383bfc51cb4c9f48026db25e05f2c0/js-replacements.json#L4-L27).

I [added a new definition](https://github.com/form8ion/renovate-config/commit/3e21b81fb62a5ef09196609c779945fb3d9c0eaa) to that file several days ago and did not get replacement PRs for the projects that I expected, despite there being several projects extending that config that still depend on `markdownlint-cli`, like https://github.com/travi/cli/blob/ca0a931f20ee3bec6628f69ba47b08603666d73e/package.json#L53 or https://github.com/travi/redux-fetch-middleware/blob/88e5787d6b0d7cd8172b6c242ed8c1a2d74ee44b/package.json#L47 as a couple of examples.

I created a minimal reproduction repo at https://github.com/travi-test/markdown-cli-replacement-reproduction, which extend only the replacements config. Assuming I didn't mess up creating the minimla example, it appears to have the same behavior of not performing the replacement.

### Relevant debug logs

i can see that the config gets loaded, but i don't find any mention of it being referenced later in the logs (but i might not know what i'm looking for)

<details><summary>Logs</summary>

```
DEBUG: massaged config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "github>form8ion/renovate-config:js-replacements"
    ]
  }
}
DEBUG: migrated config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "github>form8ion/renovate-config:js-replacements"
    ]
  }
}
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment, marked as answer]
Typo: https://github.com/form8ion/renovate-config/blob/8ca0cfcbbb383bfc51cb4c9f48026db25e05f2c0/js-replacements.json#L32

**Reply by travi** [reply]
🤦🏼 i don't know how many times i re-read that addition looking for a typo and my brain refused to see it. thanks and sorry for the noise
